src/server/routes/deploy/prebuilts/edition.ts

```diff
@@ -34,7 +34,7 @@ const requestBodySchema = Type.Object({
       description: "Version of the contract to deploy. Defaults to latest.",
     }),
   ),
-  ...txOverrides.properties,
+  ...txOverridesWithValueSchema.properties,
```

Comment:
We had `txOverrides` in contract deploy/publish end-point

I completed the implementation